src/tools/clippy/clippy_utils/src/lib.rs

```diff
@@ -869,17 +869,6 @@ pub fn is_else_clause(tcx: TyCtxt<'_>, expr: &Expr<'_>) -> bool {
     let map = tcx.hir();
     let mut iter = map.parent_iter(expr.hir_id);
     match iter.next() {
-        Some((arm_id, Node::Arm(..))) => matches!(
```

Comment:
Why can we stop handling `if let` here?

Reply:
`If let` is now a normal if` expr plus `let` plus instead of the previous `match` de-sugaring, so `if` encompasses `if let`